tests/test_engine/test_model_traffic_capture.py

```
@@ -195,6 +195,103 @@ def test_format_log_record_response_capture_fields(capture_kwargs: dict, expect_
         assert "message_content" not in row
 
 
+@pytest.mark.parametrize(
+    "capture_kwargs,expect_request_fields",
```

Ruff (0.15.21)

[warning] Wrong type passed to first argument of `pytest.mark.parametrize`; expected `tuple`

Use a `tuple` for the first argument

(PT006)